### ISSUE TITLE
Requesting new Release Version

### ISSUE THREAD

**Issue by Laurino108** [knuckleswtf/scribe#817]
### Scribe version

latest

### Your question

It would be nice to get a new version that fixes the bug I reported in #810, since #815 resolved the bug.

### Docs

- [X] I've checked [the docs](https://scribe.knuckles.wtf/laravel), the [troubleshooting guide](https://scribe.knuckles.wtf/laravel/troubleshooting), and [existing issues](https://github.com/knuckleswtf/scribe/issues?q=is%3Aissue+), but I didn't find a solution

**Comment by MrCrayon** [comment]
Until mantainer has time to release a new version you can require it like this:
```bash
composer require --dev knuckleswtf/scribe:dev-master#4811b00bf9830b66acda12509feb0ff2cd84d696
```

Instead of the last tagged version it will use master version up to that commit, it's only two commits ahead:

https://github.com/knuckleswtf/scribe/compare/4.32.0...master

**Comment by shalvah** [comment]
released 4.33.0
